Expose threadpool cleanup

RReverser opened the pull request:
Exposing threadpool cleanup from libuv + adding missing API.

While proper cleanup is not critical in Node.js, where exiting process automatically kills all associated threads, it's important for environments like Stackblitz where otherwise spawned Worker will be stuck waiting on a condvar (on an atomic) forever in background even after the Node.js command has finished, leading to Worker leaks.

toyobayashi commented:
Thank you for pointing out this, I added the public API `uv_library_shutdown`, which internally calls `uv__threadpool_cleanup`.

RReverser commented:
> Thank you for pointing out this, I added the public API `uv_library_shutdown`, which internally calls `uv__threadpool_cleanup`.

Hm I just added `uv__threadpool_cleanup` to `EXPORTED_FUNCTIONS` and called it from JS, that seemed to be enough. But I see libuv has `uv_library_shutdown` as well, so I guess makes sense to add it too.

RReverser commented:
Could you do a patch or minor release please?

toyobayashi commented:
Done.
